api/controllers.js

```diff
@@ -50,6 +50,7 @@ module.exports = {
         if (!response || response.rowCount === 0) {
           res.status(400).send('UNABLE TO GET CONVERSATION - try again later');
         } else if (response === 'noID') {
+          console.log('right here');
```

Comment:
Somewhat confusing of a log but I trust you know what it means

Reply:
leftover from tracking an error -- deleted in my next commit. Thanks!